spacy/language.py

```diff
@@ -86,6 +87,12 @@ def tokenizer_factory(nlp: "Language") -> Tokenizer:
     return tokenizer_factory
 
 
+@registry.misc("spacy.LoadLookupsData.v1")
+def load_lookups_data(lang, tables):
+    lookups = load_lookups(lang=lang, tables=tables)
```

Comment:
We could consider adding debug logging here (via `util.logger`) that mentions the tables that are loaded and that they're loaded from `spacy-lookups-data`. If the user has verbose logging enabled during training, they'll see this info (and it can potentially explain why loading takes a bit longer etc.)

Reply:
Sounds good, I'll add that.